### PR TITLE
Tag exclusions cleanup

### PR DESCRIPTION
**Pull request by vjkoskela**

### CODE REVIEW

File: app/com/arpnetworking/kairos/client/models/SamplingUnit.java [hunk 1]
```diff
@@ -53,4 +57,18 @@
...
+     * NOTE: KairosDb accepts either upper or lower case representation. This
+     * model is otherwise a pass-through but converts all enumeration values
+     * to lower case.
+     *
+     * @return json encoded value
+     */
+    @JsonValue
+    public String toJson() {
+        return name().toLowerCase(Locale.getDefault());
+    }
```
Comment (speezepearson) [inline app/com/arpnetworking/kairos/client/models/SamplingUnit.java]: nit: If Kairos isn't picky about casing, why do we need this method?

File: test/java/com/arpnetworking/testing/SerializationTestUtils.java [hunk 2]
```diff
@@ -29,7 +37,40 @@
 public final class SerializationTestUtils {
 
     // TODO(spencerpearson): allow the ObjectMapper to get passed in, if ever necessary
-    private static final ObjectMapper OBJECT_MAPPER = ObjectMapperFactory.getInstance();
+    // IMPORTANT: The configuration here must match that in MainModule for testing REST APIs
```
Comment (speezepearson) [inline test/java/com/arpnetworking/testing/SerializationTestUtils.java]: Doesn't have to be part of this diff, but: would it be feasible to refactor the ObjectMapper-instantiation out of MainModule into something that the test could import?

File: test/java/com/arpnetworking/kairos/client/models/MetricsQueryTest.java [hunk 3]
```diff
@@ -33,4 +33,28 @@ public void testTranslationLosesNothing() throws Exception {
...
+    @Test
+    public void testStartRelative() throws Exception {
+        SerializationTestUtils.assertTranslationLosesNothing(
+                ResourceHelper.loadResource(getClass(), "testStartRelative"),
+                MetricsQuery.class
+        );
+    }
+
+    @Test
+    public void tesEndRelative() throws Exception {
```
Comment (speezepearson) [inline test/java/com/arpnetworking/kairos/client/models/MetricsQueryTest.java]: nit: `test`

File: app/com/arpnetworking/kairos/service/KairosDbServiceImpl.java [hunk 4]
```diff
@@ -224,72 +202,66 @@ private static MetricsQuery useAvailableRollups(
...
                     }
-
-                    return metric;
-                }
-            }
-        })
-                .collect(ImmutableList.toImmutableList()));
-
-
-        return newQueryBuilder.build();
```
Comment (speezepearson) [inline app/com/arpnetworking/kairos/service/KairosDbServiceImpl.java]: I skimmed this and verified that it looked like you'd basically just indented all this as a side effect of switching to ThreadLocalBuilder. Let me know if you actually changed the logic inside.

File: app/com/arpnetworking/kairos/client/models/SamplingUnit.java (hunk 1 above)
Comment (vjkoskela) [reply app/com/arpnetworking/kairos/client/models/SamplingUnit.java]: You are correct, we don't. However, based on the documentation (which admittedly is not great) most of the examples were lower case. So given the decision of LC+UC=>UC or LW+UC=>LC I elected the latter which seemed more consistent with the documentation.

File: test/java/com/arpnetworking/testing/SerializationTestUtils.java (hunk 2 above)
Comment (vjkoskela) [reply test/java/com/arpnetworking/testing/SerializationTestUtils.java]: I give it a shot on the next one.

File: test/java/com/arpnetworking/kairos/client/models/MetricsQueryTest.java (hunk 3 above)
Comment (vjkoskela) [reply test/java/com/arpnetworking/kairos/client/models/MetricsQueryTest.java]: Fixed!

File: app/com/arpnetworking/kairos/service/KairosDbServiceImpl.java (hunk 4 above)
Comment (vjkoskela) [reply app/com/arpnetworking/kairos/service/KairosDbServiceImpl.java]: There were other formatting and naming changes to this block aside from the indentation; however, there were no logical changes.

Pro Tip: 
* Click the gear icon at the top of the review
* Check `Hide whitespace changes`

**Pull request merged**